shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/antlr4/imports/oracle/DDLStatement.g4

```diff
@@ -37,7 +37,7 @@ alterIndex
     ;
 
 dropTable
-    : DROP TABLE tableName
+    : DROP TABLE (schemaName DOT_)? tableName (CASCADE CONSTRAINTS)? (PURGE)?
```

Comment:
I am wondering whether need add "(schemaName DOT_)?",  as tableName definition is below:
```
tableName
    : (owner DOT_)? name
```
@tristaZero I think it's better to remove  "(schemaName DOT_)?", are you agree with me?

Reply:
Hi, If that' is the the definition of `tableName`, I agree to remove `(schemaName DOT_)?` from this `dropTable` definition.
@ThanoshanMV Could you give it a look?

Reply:
Okay, I'll revert the previous commit.